src/autoTune.ml

```diff
@@ -44,6 +44,33 @@ class functionVisitor(calling, calledBy, argLists, dynamicallyCalled) = object
     DoChildren
 end
 
+exception Found
+class findAllocsInLoops = object
+  inherit nopCilVisitor
+
+  val mutable inloop = false
+
+  method! vstmt stmt =
+    let outOfLoop stmt =
+      match stmt.skind with
+      | Loop _ -> inloop <- false; stmt
+      | _ -> stmt
+    in
+    match stmt.skind with
+    | Loop _ -> inloop <- true; ChangeDoChildrenPost(stmt, outOfLoop)
+    | _ -> DoChildren
+
+  method! vinst = function
+    | Call (_, Lval (Var f, NoOffset), args,_,_) ->
+      let desc = LibraryFunctions.find f in
+      begin match desc.special args with
```

Comment:
This gives some `exception LibraryDsl.Pattern.Expected("^::")` results on sv-benchmarksL https://goblint.cs.ut.ee/results/235-no-overflow-pr-1612-after-2/table-generator-cmp.diff.html#/table.
This needs to be adapted like #1560.

Reply:
I also added a missing `is_special` check.